src/counts.jl

```diff
@@ -19,21 +19,21 @@ end
     addcounts!(r, x, levels::UnitRange{<:Int}, [wv::AbstractWeights])
```

Comment:
```suggestion
    addcounts!(r, x, levels::UnitRange{<:Integer}, [wv::AbstractWeights])
```

Reply:
Correct. This probably belongs in a separate PR b.c. we don't change the type signature here, but I'll just commit it anyway because it's only 1 loc.